tests/tests_fabric/strategies/test_ddp.py

```diff
@@ -133,6 +133,8 @@ def test_ddp_grad_clipping(clip_type, accelerator, precision):
     "precision,expected_dtype",
     [
         (Precision(), torch.float32),
+        (HalfPrecision("16-true"), torch.float16),
+        pytest.param(HalfPrecision("bf16-true"), torch.bfloat16, marks=RunIf(bf16_cuda=True)),
```

Comment:
GPU CI only picks tests that set `min_cuda_gpus`. If you set just `bf16_cuda` then this is effectively skipped.

Could this run on CPU? If not, then you should add `min_cuda_gpus`

Same for the other tests

Reply:
It already has @RunIf(min_cuda_gpus=2) at the top, and there are other tests where it's on the CPU